src/main/kotlin/com/terraformation/backend/customer/model/PermissionRequirements.kt

```diff
@@ -537,6 +537,13 @@ class PermissionRequirements(private val user: TerrawareUser) {
     }
   }
 
+  fun rescheduleObservation(observationId: ObservationId) {
+    if (!user.canRescheduleObservation(observationId)) {
+      updateObservation(observationId)
```

Comment:
Is this the right fallback permission check? If the user doesn't have reschedule permission or update permission, then this will throw an error message of "No permission to update observation." I think this should be `readObservation` instead so it'll either throw "No permission to reschedule" or "Observation not found."

Reply:
Makes sense. Had started out that way, not sure why I overthought this. Will fix this and the other case.